docs-app/eslint.config.mjs

```diff
@@ -73,6 +73,9 @@ export default [
     plugins: {
       qunit,
     },
+    rules: {
+      ...qunit.configs.recommended.rules,
```

Comment:
Without this, none of the qunit rules are enabled (despite the fact that we included the plugin a few lines above)